packages/sanes-chrome-extension/src/extension/background/model/signingServer/index.unit.spec.ts

```diff
@@ -25,27 +23,32 @@ const buildGetIdentitiesRequest = (method: string, customMessage?: string): Json
 
 withChainsDescribe('background script handler for website request', () => {
   let db: StringDb;
+  let persona: Persona;
+  let signingServer: SigningServer;
 
   beforeAll(() => {
     jest.spyOn(txsUpdater, 'transactionsUpdater').mockImplementation(() => {});
-    jest.spyOn(txsUpdater, 'requestUpdater').mockImplementation(() => {});
+    jest.spyOn(txsUpdater, 'updateRequestProvider').mockImplementation(() => {});
   });
   beforeEach(async () => {
     localStorage.clear();
     db = createMemDb();
-    await createPersona(db, 'test password', undefined);
+    signingServer = new SigningServer();
+    persona = await Persona.create(db, signingServer, 'test-password', undefined);
+    signingServer.start(persona.getCore());
   });
   afterEach(() => {
-    clearPersona();
+    persona.destroy();
+    signingServer.shutdown();
     db.close();
   });
   afterAll(() => {
     jest.spyOn(txsUpdater, 'transactionsUpdater').mockReset();
-    jest.spyOn(txsUpdater, 'requestUpdater').mockReset();
+    jest.spyOn(txsUpdater, 'updateRequestProvider').mockReset();
   });
 
   function checkNextRequest(request: JsonRpcRequest, sender: string): void {
-    const req = RequestHandler.next();
+    const req = signingServer['requestHandler'].next();
```

Comment:
didn't you just tell Abel to not access private members? Usually accessing private data should not be necessary to test the bahaviour of a class

Reply:
This is a fair and good comment. I forgot to put the doc message. Unfortunately RequestHandler now it is well encapsulated under SigningServer and for obtaining the next request under a "unit-test", this is the most efficient and simplest way to do it.

Can you figure out an alternative?